### PR TITLE
eslint rule for connectionFromArray import restriction, resolves #53

### LINKED ISSUE #53: Use custom connectionFromArray everywhere
> Could you please use [this Eslint rule](https://eslint.org/docs/rules/no-restricted-imports) to make sure we'll use it everywhere even in the future? See for example this: https://github.com/kiwicom/react-native-app/blob/e39ea6a773dfffc14180edce4cfb7e1caa4b0b71/.eslintrc.js#L40-L74

### PR DESCRIPTION
**Pull request by Rulsky**
This is a PR for a #53 with addition of eslint rule to restrict importing native function `connectionFromArray` from `graphql-relay` to encourage usage of in-house implementation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,13 +24,13 @@
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-core": "^6.25.0",
-    "babel-eslint": "^7.2.3",
+    "babel-eslint": "~8.2.2",
```
Comment (mrtnzlml) [inline package.json]: Why did you decide to change it from `^` to `~`? Is there a good reason for that? Can we please stick with the [caret range](https://yarnpkg.com/lang/en/docs/dependency-versions/#toc-caret-ranges)?

Comment (Rulsky) [reply package.json]: I copied exact version from kiwicom/react-native-app. Will fix it.

**Comment by Rulsky** [comment]
Changed a version range for babel-eslint

File: package.json (hunk 1 above)
Comment (mrtnzlml) [reply package.json]: Haha, now the question is why do we have it in our RN repo. Thanks for pointing this out... :)

**Comment by mrtnzlml** [comment]
@Rulsky You stated [in this comment](https://github.com/kiwicom/graphql/issues/53#issuecomment-383059779) that there should be few changes in other files but they are not part of this PR. But this is basically what is this PR about so you have to fix this as well otherwise the tests are going to fail (and therefore we cannot merge it)... 🙂

**Comment by mrtnzlml** [comment]
Also, we prefer clean git history of PRs at any time so you should squash your changes into one commit and rebase it on top of the master branch. However, if you don't know how to do that I can fix it for you... 😉

**Comment by Rulsky** [comment]
updated consumers of `connectionFromArray`, squashed into one commit, CI passed.

**Comment by mrtnzlml** [comment]
@Rulsky Thank you very much! :)

**Comment by Rulsky** [comment]
Thanks for the opportunity

**Pull request merged**